docs/architecture/blazor-for-web-forms-developers/introduction.md

```diff
@@ -8,6 +8,50 @@ ms.date: 09/11/2019
+
+## WebAssembly to the rescue!
+
+In 2015, the major browser vendors joined forces in a W3C Community Group to create a new open web standard called WebAssembly. WebAssembly is a byte code for the Web. The idea is that if you can compile your code to WebAssembly, it can then run on any browser on any platform at near native speed. Initial efforts focused on C/C++, resulting in some pretty dramatic demonstrations of getting native 3D graphics engines running directly in the browser without any plugins. WebAssembly has since been standardized and implemented by all major browsers. Work on running .NET on WebAssembly was announced in late 2017 and is expected to ship in 2020, including support from .NET 5. The ability to run .NET code directly in the browser enables full-stack web development with .NET.
+
+## Blazor: full-stack web development with .NET
+
+But just being able to run .NET code in a browser doesn't give you an end-to-end experience building client-side web apps. That's where Blazor comes in. Blazor is a new client-side web UI framework based on C# instead of JavaScript that can run directly in the browser via WebAssembly. No browser plugins are required. Alternatively, Blazor apps can run server-side on .NET Core and handle all user interactions over a real-time connection with the browser. Blazor includes a full UI component model and built-in facilities for forms and validation, dependency injection, client-side routing, layouts, in-browser debugging, and JavaScript interop. Blazor also comes with great tooling support in Visual Studio and Visual Studio Code.
+
+Blazor shares a lot in common with ASP.NET Web Forms: both frameworks are component-based, event-driven, stateful UI programming models. The main architectural difference is that ASP.NET Web Forms runs on the server, while Blazor can run on the client in the browser. But if you're coming from an ASP.NET Web Forms background, there's a lot in Blazor that will feel very familiar. Blazor is a natural solution for ASP.NET Web Forms developers looking for a way to take advantage of client-side development and the open-source and cross-platform future of .NET.
+
+This book gives an introduction to Blazor that is catered specifically to ASP.NET Web Forms developers. Each Blazor concept is presented in the context of analogous ASP.NET Web Forms features and practices. By the end of this book, you should have a solid understanding of how to build Blazor apps, how Blazor works, how it ties in with .NET Core, and also some reasonable strategies for migrating existing ASP.NET Web Forms apps to Blazor where appropriate.
+
+## Getting started with Blazor
+
+Getting started with Blazor is easy. Go to <https://blazor.net> and follow the links to install the appropriate .NET Core SDK and Blazor project templates. You'll also find instructions for setting up the Blazor tooling in Visual Studio or Visual Studio Code.
```

Comment:
What's the tooling story for VS for Mac?

Reply:
Since this book is targeted at Web Forms developers, I think it's reasonable to focus on the Windows + Visual Studio experience